src/app/common/components/navbar/common-components-navbar.component.scss

```diff
@@ -27,6 +27,7 @@
   .navigation-link {
     text-decoration: none;
     color: colors.$pure-black;
+    padding-left: 15px;
```

Comment:
In that case you should use margin I think. It will add space outside of the element instead inside.